ctapipe/io/pointing.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any
+
+import astropy.units as u
+import numpy as np
+import tables
+from scipy.interpolate import interp1d
+
+from ctapipe.core import Component, traits
+
+from .astropy_helpers import read_table
+
+
+class PointingInterpolator(Component):
+    bounds_error = traits.Bool(default_value=True).tag(config=True)
+    extrapolate = traits.Bool(default_value=False).tag(config=True)
```

Comment:
These traits need help strings, especially as the value of one will silently override the value of the other. Ie you if you set both `bounds_error` and `extrapolate` to True then `bounds_error` will silently win the conflict.